.pylintrc

```diff
@@ -1,5 +1,7 @@
 [MASTER]
```

Comment:
**question**: any purpose to have such a long pylintrc over using the 5 line one from agrc/python?

Reply:
Because I manually merged what was in agrc/python to my existing .pylintrc. I intend to go through and remove all the commented-out stuff to see exactly what the differences are and if they matter, I just haven't gotten to it yet.